### PR TITLE
Update DLK -> Converter docs

### PR DESCRIPTION
**Pull request by Joeper214**
## What this patch does to fix the issue.
- Remove DLK strings, contents and installation
## Link to any relevant issues or pull requests.
n/a

### CODE REVIEW

File: docs/dlk/archive_file.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ make ar_fpga
 
 These commands will generate the following files:
 ```
-libdlk_x86.a
-libdlk_arm.a
-libdlk_fpga.a
+lib_x86.a
```
Comment (iizukak) [inline docs/dlk/archive_file.md]: In #724, @primenumber change library name. `libdlk_x86` is correct.

Comment (Joeper214) [reply docs/dlk/archive_file.md]: Oh. Thanks for pointing that out. I'll revert these changes.

File: docs/converter/shared_library.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ After you generate your project from your protocol buffer, you can build librari
 The compile command would be like below.
 
 ```
-make lib_x86
+make libdlk_x86
```
Comment (primenumber) [inline docs/converter/shared_library.md]: target name of make is not changed.

Comment (Joeper214) [reply docs/converter/shared_library.md]: oh. Thanks for noticing that! I'll revert this change.

**Comment by iizukak** [comment]
@primenumber Please check your commented issue.

**Pull request merged**